packages/core/src/components/axes/axis.ts

```diff
@@ -415,7 +430,7 @@ export class Axis extends Component {
 
 		// truncate the label if it's too long
 		// only applies to discrete type
-		if (!isTimeScaleType && axisOptions.scaleType) {
+		if (truncationTypeProvided && truncationTypeProvided !== "TruncationTypes.NONE" && !isTimeScaleType && axisOptions.scaleType) {
```

Comment:
truncationTypeProvided isn't necessary.

we can set truncation by default to `TruncationTypes.NONE` or any other types (I don't suggest defaulting to none, maybe END_LINE by default?

then we also don't need to check whether it's been provided or not